http.go

```diff
@@ -417,7 +432,11 @@ func httpPutViolationsInner(w http.ResponseWriter, r *http.Request, typestr stri
 		}
 		exc := false
 		if rep.Type == "ip" {
-			exc = isException(rep.Object)
+			if exc, err = isException(rep.Object); err != nil {
+				log.Warnf(err.Error())
+				w.WriteHeader(http.StatusInternalServerError)
```

Comment:
As with put reputation, since this is just for logging should just log the warning and not return a `500`